content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md

```diff
@@ -62,6 +62,21 @@ After the above port-forwarding is active, the Hubble UI can be shown by navigat
 
 Please note that to have the Hubble addon available, the KKP installation has to be configured with `hubble` as [an accessible addon]({{< relref "../../../architecture/concept/kkp-concepts/addons/#accessible-addons" >}}).
 
+The following table lists the supported operating systems and cloud providers for cilium CNI in KKP:
+
+|   | Ubuntu | Flatcar | RHEL | Amazon Linux 2 |
+|---|---|---|---|---|
+| AWS | ✓ | ✓ | ✓ | ✓ |
+| Azure | ✓ | ✓ | ✓ | x |
+| Digitalocean  | ✓ | x | x | x |
+| Google Cloud Platform | ✓ | x | x | x |
```

Comment:
If some OS is not available on a given platform (E.g. Amazon Linux on GCP), I would propose using `-` or `n/a` instead of `x` to make it more clear.

Reply:
Great point, updated.